collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -112,6 +109,16 @@ sherlock() {
               FOUND="rke"
           fi
       fi
+      if $(command -v kubeadm >/dev/null 2>&1)
+        then
+          if $(kubeadm version >/dev/null 2>&1)
+            then
+              DISTRO=kubeadm
+              echo "kubeadm"
+            else
+              FOUND+="kubeadm"
+          fi
+      fi
```

Comment:
As discussed in https://github.com/rancherlabs/support-tools/pull/213, let's not have auto-detection enabled.

```suggestion
```